src/layer/__tests__/WmsLayer.ts

```diff
@@ -36,3 +37,45 @@ test('WmsLayer.getMapUrl returns an URL', () => {
   expect(imageUrl).not.toHaveQueryParams(['showlogo']);
   expect(imageUrl).not.toHaveQueryParams(['transparent']);
 });
+
+test('WmsLayer.getMap makes an appropriate request', () => {
+  const bbox = new BBox(CRS_EPSG4326, 19, 20, 20, 21);
+  const layerId = 'PROBAV_S1_TOA_333M';
+  const layer = new WmsLayer(
+    'https://proba-v-mep.esa.int/applications/geo-viewer/app/geoserver/ows',
+    layerId,
+  );
+
+  const getMapParams = {
+    bbox: bbox,
+    fromTime: new Date(Date.UTC(2020, 1 - 1, 10, 0, 0, 0)), // 2020-01-10/2020-01-10
+    toTime: new Date(Date.UTC(2020, 1 - 1, 10, 23, 59, 59)),
+    width: 512,
+    height: 512,
+    format: MimeTypes.JPEG,
+  };
+  layer.getMap(getMapParams, ApiType.WMS);
+
+  expect(axios.get.mock.calls.length).toBe(1);
```

Comment:
this one could be replaced with
```
expect(axios.get).toHaveBeenCalledTimes(1);
```

Reply:
It does look a bit easier to understand, so I have replaced it, thx.